### PR TITLE
Sass-feature

### PR DESCRIPTION
**Pull request by falguniislam**
Description:
We created our reset CSS, a fixed number of grid which is separated by gutters. We applied the framework to our project. We identify the web page which we would like clone and break it down into its measure element like in the previous project. We use our frameworks to layout the structure of the page.

### CODE REVIEW

File: framework.css [hunk 1]
```diff
@@ -0,0 +1,1807 @@
...
+
+.rounded-2 {
+    border-radius: 2rem;
+}
+
+.circle {
+    border-radius: 50%;
+}
+
+/*# sourceMappingURL=custom.css.map */
```
Comment (whiz25) [inline framework.css]: remove this comment and any other comments in your code.

File: framework.css [hunk 2]
```diff
@@ -0,0 +1,1807 @@
...
+    grid-column: span 4;
+}
+
+@media (max-width: 1024px) {
+    .col-lg-4 {
+        -ms-grid-column: 1;
+        -ms-grid-column-span: 12;
+        grid-column: 1 /span 12;
+    }
+}
```
Comment (whiz25) [inline framework.css]: combine these into one query;
#### media (max-width: 1024px) {
.col-lg-3 {}
.col-lg-4{}
.col-lg-5{}
#### }
This makes your code clean and easy to maintain.

File: framework.css (hunk 1 above)
Comment (falguniislam) [reply framework.css]: because i my coding partner wants to work alone.

**Comment by ebukaume** [comment]
Hi @falguniislam :wave: 

Nice work so far. I really like your work, your clone looks so sweet.

However, there is a problem. I noticed you used a different sticker configuration than the one provided for you.

https://github.com/falguniislam/SASS-Design-your-own-Grid-based-Framework/blob/9dd31f4ab479a37a78866f5f54e7a5114c8e0305/.stickler.yml#L1-L11

In this configuration, you basically ignored all your CSS file. In other words, you set it to pass no matter the content. One would then ask why you used stickler in the first place. :man_shrugging: 

Kindly, use the configuration specified [here](https://github.com/microverseinc/linters-config/tree/master/css).

**Please submit another code review request after fixing the issues.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**